testcase/build_once.mk

```diff
@@ -1,7 +1,13 @@
 # expect protoc compile/link only once.
+
+# Caveat: this test relies on Make and Ninja updating 
```

Comment:
I don't quite understand this comment; what does "updating the targets in the same order" mean?